config.json

```diff
@@ -28,7 +28,7 @@
 
   },
   "classify": {
-    "path": "modules/shared/data/MatrixToClassify/matrix_to_classify"
+    "path": "modules/shared/data/MatrixToClassify/matrix_to_classify",
 	"network_path": "modules/shared/data/NeuralNetwork/",
```

Comment:
now there is one to many. 
Please also fix the intendation "path" is on the samel level as "network_path", same goes for the part earlier in the file.